### ISSUE TITLE
make a dynamic sitemap

### ISSUE THREAD

**Issue by bobby365** [Sank6/Discord-Bot-List#188]
it would be nice to have a dynamic site map so when bots are submitted and you have your website listed up on Google they will be listed alongside with your website

**Comment by Zaid-maker** [comment]
That's would be nice

**Comment by Sank6** [comment]
Yeah seems like a good idea. I don't think it would work that well with google, because their crawler probably doesn't constantly check the sitemaps. I'll add it to the readme though.

**Comment by idk-pixel** [comment]
@Sank6 to get ur web listed u have the user has to do it watch this video: https://youtu.be/rpYm3vpIKtg

**Comment by discord-bot-purchases** [comment]
@Sank6
